.github/workflows/npm-package-release.yml

```
@@ -74,23 +74,42 @@ jobs:
           git add "packages/orderbook/package.json"
           git add "packages/ui-components/package.json"
           git add "package-lock.json"
-          git commit -m "NPM Package Release ${{ env.NEW_VERSION }}"
-          git tag ${{ env.NEW_VERSION }}
+          git commit -m "NPM Package Release v${{ env.NEW_VERSION }}"
+          git tag npm-v${{ env.NEW_VERSION }}
 
       # Push the commit to remote
       - name: Push Changes To Remote
         run: |
           git push origin 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)